### PR TITLE
WB-1069 - Add the Final Set of Props to LabeledTextField

### PR DESCRIPTION
**Pull request by RivasCVA**
## Summary:
Adding the final set of props to `LabeledTextField`.

This includes:
* placeholder
* light
* style
* testId

Included unit tests to verify these props work as expected (Coverage is 100%).

Issue: https://khanacademy.atlassian.net/browse/WB-1069

## Test plan:
1. Open Styleguidist or React Storybook (See generated links).
2. View the new examples under Form / LabeledTextField.

---

### Light Prop Example
labeled-text-field-light-example

### Style Prop Example
labeled-text-field-style-example

### CODE REVIEW

File: packages/wonder-blocks-form/src/components/__tests__/labeled-text-field.test.js [hunk 1]
```diff
@@ -253,4 +254,64 @@ describe("LabeledTextField", () => {
...
+        });
+
+        // Act
+        const wrapper = mount(
+            <LabeledTextField label="Label" style={styles.style1} />,
+        );
+
+        // Assert
+        const textField = wrapper.find("TextField");
+        expect(textField).toHaveStyle(styles.style1);
```
Comment (jandrade) [inline packages/wonder-blocks-form/src/components/__tests__/labeled-text-field.test.js:301]: Nice!

File: packages/wonder-blocks-form/src/components/labeled-text-field.js [hunk 2]
```diff
@@ -110,44 +132,42 @@ export default class LabeledTextField extends React.Component<Props, State> {
 
     handleValidation: (errorMessage: ?string) => mixed = (errorMessage) => {
         const {onValidation} = this.props;
-        this.setState({error: errorMessage});
-        if (onValidation) {
-            onValidation(errorMessage);
-        }
+        this.setState({error: errorMessage}, () => {
```
Comment (jandrade) [inline packages/wonder-blocks-form/src/components/labeled-text-field.js:135]: Is this change related to the comments in the previous PR, right?

**Review by jandrade — APPROVED** [review]
Changes look great to me, thanks!

File: packages/wonder-blocks-form/src/components/labeled-text-field.js (hunk 2 above)
Comment (RivasCVA) [reply packages/wonder-blocks-form/src/components/labeled-text-field.js:135]: Correct!

File: packages/wonder-blocks-form/src/components/labeled-text-field.js [hunk 3]
```diff
@@ -203,8 +203,8 @@ export default class LabeledTextField extends React.Component<Props, State> {
                                 value={this.state.value}
                                 placeholder={placeholder}
                                 disabled={disabled}
-                                validation={validation}
-                                onValidation={this.handleValidation}
+                                validation={validate}
+                                onValidation={this.handleValidate}
```
Comment (RivasCVA) [inline packages/wonder-blocks-form/src/components/labeled-text-field.js:207]: I will be updating the props here when I rename them to `validate` and `onValidate` in `TextField` as well (See this [ticket](https://khanacademy.atlassian.net/browse/WB-1071)).

**Pull request merged**